Examples.fshx

```diff
@@ -38,11 +50,79 @@ InstanceOf: CancerPatient
 * communication.language = urn:ietf:bcp:47#en-US "English (Region=United States)"
 * communication.language.text = "English"
 
+
+Instance: mCODEPatientExample02
+InstanceOf: CancerPatient
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerPatient"
+* extension[us-core-race].extension[ombCategory].valueCoding = urn:oid:2.16.840.1.113883.6.238#2186-5 "Not Hispanic or Latino"
+* extension[us-core-race].extension[text].valueString = "Not Latino"
+* extension[us-core-birthsex].valueCode.system = "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender"
```

Comment:
Suggest implementing this by defining an alias, e.g., 
Alias: GENDER = "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender"
Removing line 59 where you set the system...
Modifying line 60 as:
* extension[us-core-birthsex] = GENDER#F